src/test/java/core/basesyntax/strategy/BalanceOperationServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BalanceOperationServiceTest {
+    @Test
+    public void process_balanceOperationServiceValidData_Ok() {
+        Fruit fruit = new Fruit("Banana");
+        FruitTransaction fruitTransaction = new FruitTransaction();
+        StorageDao storageDao = new StorageDaoImpl();
+
+        fruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        fruitTransaction.setFruit(fruit);
+        fruitTransaction.setQuantity(10);
+
+        OperationService operationService = new BalanceOperationService(storageDao);
+
+        operationService.process(fruitTransaction);
+
+        Assert.assertTrue(Storage.fruitStorage.containsKey(fruit));
+        Assert.assertTrue(Storage.fruitStorage.containsValue(10));
```

Comment:
достатньо робити одну таку перевірку
```suggestion
        Assert.assertEquals(10, Storage.fruitStorage.get(fruit));
```